프로그래머스/1/12982. 예산/예산.js

```diff
@@ -0,0 +1,13 @@
+function solution(d, budget) {
+    const amountArr = d.slice().sort((a, b) => a - b);
+    let balance = budget;
+    let count = 0;
+    
+    for(let i = 0; i < amountArr.length; i++) {
+        if(balance - amountArr[i] >= 0) {
+            balance = balance - amountArr[i];
+            count ++;
+        }
```

Comment:
```suggestion
            count++;
        } else {
            break; 
        }
```
예산을 초과하는 순간 반복을 중단하는 코드를 넣어보는건 어떨까용 ㅎㅎ